Requests/Builders/CreateMemberRequestBuilder.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pyrus.ApiClient.Requests.Builders
+{
+   public  class CreateMemberRequestBuilder
+    {
+        private readonly CreateMemberRequest _request;
+
+        public CreateMemberRequestBuilder(string email)
+        {
+            _request = new CreateMemberRequest
+            {
+                Email = email
+            };
+        }
+
+		public CreateMemberRequestBuilder WithFirstName(string name)
+		{
+			_request.FirstName = name;
+			return this;
+		}
+
+		public CreateMemberRequestBuilder WithLastName(string name)
+		{
+			_request.LastName = name;
+			return this;
+		}
+
+		public CreateMemberRequestBuilder WithEmail(string email)
```

Comment:
This class have a single costrcutor with email parameter. Having another method to override constructor parameter will lead to a confusion